config.py

```diff
@@ -10,6 +10,7 @@
 from flask_babel import lazy_gettext as _
 
 from facets import FacetConstants
+from sqlalchemy.exc import ArgumentError
```

Comment:
An unrelated change to add a missing import. This error is caught if your database environment variable is in the wrong format.